src/FileSystem-Tests-Core.package/FileSystemTest.class/instance/testCreateFile.st

```diff
@@ -0,0 +1,8 @@
+tests
+testCreateFile
+	| path |
+	path := FileSystem workingDirectory / 'plonk' / 'griffles'.
+	(FileSystem workingDirectory / 'plonk') ensureCreateDirectory.
```

Comment:
Maybe, the truth is that we should test both memory and non memory file systems to be entirely sure.

In any case, the `#tearDown` seems to delete paths that were registered for cleanup.